docs/source/modin/series.rst

```diff
@@ -216,5 +214,14 @@ Series
     Series.str
     Series.dt
 
+.. rubric:: Snowflake Specific
```

Comment:
Can we change the order and put this first? I would put the new/deviating APIs first so readers see them immediately. I assume they tend to know pandas, so we need to make sure these get visibility.

Reply:
Hmm - I'm not sure if this is wise. I imagine that we want to document the pandas APIs first to mimic the pandas docs as much as possible. cc: @sfc-gh-mvashishtha @sfc-gh-yzou